samcli/lib/utils/git_repo.py

```diff
@@ -118,8 +122,8 @@ def clone(self, clone_dir: Path, clone_name: str, replace_existing: bool = False
                 temp_path = os.path.normpath(os.path.join(tempdir, clone_name))
                 git_executable: str = GitRepo._git_executable()
                 LOG.info("\nCloning from %s", self.url)
-                subprocess.check_output(
-                    [git_executable, "clone", self.url, clone_name],
+                check_output(
+                    [git_executable, "clone", "--shallow", self.url, clone_name],
```

Comment:
What's `--shallow`? Never seen or used it, nor can I find it in the docs: https://git-scm.com/docs/git-clone

Reply:
removed, I was using a non-existing flag..